packages/components/src/components/checkbox-group/checkbox-group.stories.ts

```diff
@@ -1,16 +1,19 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
 
 const { argTypes, parameters } = storybookDefaults('sd-checkbox-group');
 const { generateTemplate } = storybookTemplate('sd-checkbox-group');
 const { overrideArgs } = storybookHelpers('sd-checkbox-group');
 
+/**
+ * Checkbox group allows users to select multiple options from a list of options.
```

Comment:
Allows users to select multiple options from a list of options.

Reply:
Updated

Reply:
You sure? I still see "Checkbox group allows users to select multiple options from a list of options." instead of "Allows user to select multiple options from a list of options."

Reply:
This change had not been pushed. Good catch, thank you